cmd/keytransparency-client/grpcc/grpc_client.go

```diff
@@ -26,6 +26,10 @@ import (
 	"log"
 	"time"
 
+	"github.com/golang/protobuf/proto"
```

Comment:
we are putting 3rd party imports below the key transparency and trillian imports